pumps/hybrid.go

```diff
@@ -128,8 +130,25 @@ func (p *HybridPump) Init(config interface{}) error {
 			p.trackAllPaths = trackAllPaths.(bool)
 		}
 
-		if storeAnalyticPerMinute, ok := meta["store_analytics_per_minute"]; ok {
-			p.storeAnalyticPerMinute = storeAnalyticPerMinute.(bool)
+		if storeAnalyticsPerMinute, ok := meta["store_analytics_per_minute"].(bool); ok {
+			p.storeAnalyticsPerMinute = storeAnalyticsPerMinute
+			p.aggregationTime = 1
+		} else {
+			aggregationTime, ok := meta["aggregation_time"].(int)
```

Comment:
should we validate as well if this is nil?

Reply:
Same as above. Checking the "ok" value is not a way to validate if aggregationTime is nil?

Reply:
done!